### PR TITLE
Deny to resolve package directly.

### PR DESCRIPTION
**Pull request by proofit404**

### CODE REVIEW

File: tests/test_injector.py [hunk 1]
```diff
@@ -400,7 +400,6 @@ class Foo(Injector):
     assert "x" in dir(Foo(x=1))
 
 
-@pytest.mark.xfail()
```
Comment (proofit404) [inline tests/test_injector.py:403]: Outdated `# pragma: no cover` should be removed as well.

File: tests/helpers/examples/submodule.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
+variable = 1
+
+
+def function():
```
Comment (proofit404) [inline tests/helpers/examples/submodule.py:4]: We should explicitly deny `Container.function` resolve.

File: tests/helpers/examples/submodule.py [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+    """Return dummy data."""
+    return 1
+
+
 class Foo:
```
Comment (proofit404) [inline tests/helpers/examples/submodule.py:9]: We should explicitly allow `Container.foo` resolve.

**Comment by 0cry** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Only issue marked as backward incompatible is allowed to have breaking changes</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9a335b755abbc53ad6f59a13f506145a7f71e5a1
</p>

**Pull request merged**